Vivarium/Assets/Scripts/Player/PlayerController.cs

```diff
@@ -133,6 +135,8 @@ private void DeselectAction()
         actionViewer.HideAction();
         _actionIsSelected = false;
         _selectedAction = null;
+        UIController.Instance.MoveRangeText.text = null;
+        UIController.Instance.DamageText.text = null;
```

Comment:
Similar issue to the one I noted above. I would make a `ClearActionStats` function in `UIController` and call it here.